### DISCUSSION TITLE
The more I discover Quarto, the more ... wow !!!

### DISCUSSION THREAD

**Discussion by cavo789** [quarto-dev/quarto-cli#8411] [Show and Tell]
### Description

The most I'm reading here (on github - discussions) or there (google "Quarto" "tips" "tutorials" ...), the most I'm saying to myself : Wow ! It's just incredible what we can do with Quarto.

I'm a lover of Markdown, using it since years and converting my notes to pdf, html, reveal.js, ... using self-made scripts or pandoc or revealmd or ... but, that's crazy, with just Quarto and the correct settings (_quarto.yml, _variables.yml, environment files, gitlab CI / github actions, ...) it's possible to do so many crazy things. Just using Quarto (even if quarto is using other tools; we just need to learn Quarto).

Create own filters (lua) is not so difficult; create own templates is really piece of cake (mine is based on Eisvogel for PDF); ...

I just would like to open this discussion **to thank the people behind Quarto, who are giving up their time to develop the tool and to respond here so quickly and accurately**.

Thank you! You rocks!

**Comment by cderv** [comment]
Thanks a lot for the feedback ! 

That means truly a lot for us @quarto-dev/developers working hard on making Quarto a good product !

**Comment by cscheid** [comment]
We definitely appreciate the kind words!

**Reply by cavo789** [reply]
You, and the team, desserve it. Great work !

**Comment by sda030** [comment]
Not to mention the struggle with upstream management (Pandoc, knitr, Typst, gt, R, Python, etc) and their bugs, intricasies, inconsistencies and moving targets... All open source.

**Comment by harrelfe** [comment]
Quarto has been a game-changer since its first release.  Now that it is fully developed, wow!  And the development team constantly improves it.

**Comment by GuillaumeChretien** [comment]
Hello everyone, I'd like to take this opportunity to sincerely thank the team! It's a pleasure to work in this environment that, without much fanfare, creates bridges between multiple technologies, which also helps to bring colleagues closer together who were previously confined to a single language or tech! A special thanks for the integration of Observable, which gives JavaScript enthusiasts an excellent playground!
